src/Root.tsx

```diff
@@ -0,0 +1,20 @@
+import { Route, HashRouter as Router, Routes } from 'react-router-dom';
+import { App } from './App';
+import { HomePage } from './pages/HomePage';
+import { PeoplePage } from './pages/PeoplePage';
+import { NotFoundPage } from './pages/NotFoundPage';
+
+export const Root = () => (
+  <Router>
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route index element={<HomePage />} />
+        <Route path=":home" element={<HomePage />} />
```

Comment:
The route on line 12 seems redundant since the `index` route on line 11 already handles the root path with `HomePage`. Consider removing the `:home` path unless it serves a specific purpose.